helpers/helpers-spec_service.bal

```diff
@@ -178,4 +189,27 @@ service / on main_endpoint {
 
     }
 
+    resource function post currency/rate(curreny_converter_payload payload) returns curreny_converter_responseOk|Error_responseBadRequest|error {
+
+        float? fromRate = RATES_TO_USD[payload.fromCurrency];
```

Comment:
Since payload.fromCurrency might be accessed multiple time, could extract to a variable. Can also do in one go

```ballerina
curreny_converter_payload {fromCurrency, toCurrency} = payload;
```

Reply:
Fixed with 6c351e3